shell/src/main/java/tachyon/shell/TfsShell.java

```diff
@@ -138,8 +138,8 @@ public int run(String... argv) {
     try {
       command.run(args);
       return 0;
-    } catch (IOException ioe) {
-      System.out.println(ioe.getMessage());
+    } catch (IOException e) {
+      System.out.println(e.getMessage());
```

Comment:
I agree, stack trace in the standard output is not a good thing. I wonder if it would make sense to generate the stack trace into a log file though?